application/test/process/script_launcher_test.rb

```diff
@@ -116,10 +122,13 @@ class ScriptLauncherTest < ActiveSupport::TestCase
     @launcher.stubs(:elapsed_string).with(Time.parse('2025-08-28T14:30:45')).returns('00:02:00')
     @launcher.stubs(:now).returns('2025-08-28T14:32:45')
 
+    expected_log_filename = 'launch_detached_process-35-W2025.log'
```

Comment:
I would use the right file name `launch_detached_process-2025-W35.log` for these tests too to avoid confusion